wandb/lib/server.py

```diff
@@ -16,8 +16,8 @@ class Server(object):
     def __init__(self, api=None):
         self._api = api or InternalApi()
         self._error_network = None
-        self._viewer = None
-        self._flags = None
+        self._viewer = {}
+        self._flags = {}
```

Comment:
There are places in the code that assume these are set, I think this is the right thing to do.